cmake/test/test_export/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.20.1)
+project(libcudacxxTestExport CXX)
+
+message(STATUS "libcudacxx_DIR=${libcudacxx_DIR}")
+find_package(libcudacxx)
```

Comment:
Any reason to not mark this as `REQUIRED`?

Reply:
Good catch, it should be. I'll update this.